R/module_transform_data.R

```diff
@@ -68,17 +68,22 @@ srv_transform_data <- function(id, data, transforms, modules) {
 
   moduleServer(id, function(input, output, session) {
     logger::log_debug("srv_teal_data_modules initializing.")
-    Reduce(
+    is_transformer_failed <- reactiveValues()
+    transformed_data <- Reduce(
```

Comment:
```suggestion
    Reduce(
```

Reply:
This goes together with this https://github.com/insightsengineering/teal/pull/1341/files#r1756590831